client/src/constants/Fonts.tsx

```diff
@@ -1,11 +1,11 @@
 import { css } from 'styled-components';
 
 const FONTS_MAIN = css`
-  font-family: Optima, Verdana, Arial, Times;
+  font-family: Optima, Verdana, Arial, sans-serif;
 `;
 
 const FONTS_SECONDARY = css`
-  font-family: Baskerville, Optima, Arial, Times;
+  font-family: Baskerville, Optima, Arial, sans-serif;
```

Comment:
I looked at all of the fonts and I think we should swap out Baskerville for another sans-serif font. Baskerville is the only serif font in these and it sticks out

Reply:
i changed the font to century gothic which is a sans-serif and i think im in love